docs/docs/responses.mdx

```diff
@@ -257,58 +212,52 @@ responses:
             text: Select a date
 ```
 
-
 ## Using Responses in Conversations
 
 ### Calling Responses as Actions
 
-1. So long as the name of the response starts with `utter_`, the response can
-   directly be used as an action. You would add the response
-   to the domain:
+So long as the name of the response starts with `utter_`, the response can
+directly be used as an action. Say you have the following response in your domain:
 
 ```yaml-rasa title="domain.yml"
-   responses:
-     utter_greet:
-     - text: "Hey! How are you?"
-   ```
+responses:
+  utter_greet:
+  - text: "Hey! How are you?"
+```
 
-   Afterwards, you can use the response as an action in the
-   stories:
+You can use that same response as an action in your stories:
 
-```yaml-rasa title="domain.yml"
-   stories:
-   - story: greet user
-     steps:
-     - intent: greet
-     - action: utter_greet
+```yaml-rasa {5} title="stories.yml"
+stories:
+- story: greet user
+  steps:
+  - intent: greet
+  - action: utter_greet
 ```
 
-When `utter_greet` is run as an action, it will send the message from
+When `utter_greet` action runs, it will send the message from
 the response back to the user.
 
 :::note
-If you want to change the text, or any other part of the bots response,
+If you want to change the text, or any other part of the response,
 you need to retrain the assistant before these changes will be picked up.
-
 :::
 
 ### Calling Responses from Custom Actions
```

Comment:
Can we clarify this is using SDK and also show how to do it in another action server?